packages/mgt/package.json

```diff
@@ -35,7 +35,7 @@
     "compile:watch": "tsc -w",
     "lint": "eslint -c ../../.eslintrc.js 'src/**/*.ts'",
     "postpack": "cpx *.tgz ../../artifacts",
-    "rollup": "rollup -c",
+    "rollup": "rollup -c --bundleConfigAsCjs",
```

Comment:
nope, we're not shipping cjs any longer, the output of this project is intended to be esm

Reply:
Oh, it's saying that our config file is esm and needs to converted to cjs for Rollup to handle it. Let's move the config to a `.mjs` file instead